nirc_ehr/resources/queries/dbo/q_demographics.sql

```diff
@@ -1,14 +1,37 @@
 
-SELECT ANIMAL_ID_NUMBER AS Id,
-       BIRTH_DATE,
-       GENDER_ID AS gender,
-       SSB_ID AS species,
-       BIRTH_DATE AS birth,
-       DEATH_DATE AS death,
-       ANIMAL_DISPOSITION_ID AS disposition,
-       DESCRIPTION AS description,
-       TERMINATION_REASON_ID AS termination,
-       IMPLANT_NUMBER AS implantNumber,
-       VENDOR_ANIMAL_NUMBER AS vendorNumber,
-       ACTIVE_YN AS status
-FROM Animal
\ No newline at end of file
+SELECT anm.ANIMAL_ID_NUMBER AS Id,
+       anm.BIRTH_DATE,
+       anm.GENDER_ID AS gender,
+       anm.SSB_ID.SPECIES_ID AS species,
+       anm.BIRTH_DATE AS birth,
+       anm.DATE_WEANED AS weanedDate,
+       anm.DEATH_DATE AS death,
+       anm.ANIMAL_DISPOSITION_ID AS disposition,
+       anm.DESCRIPTION AS description,
+       anm.TERMINATION_REASON_ID AS termination,
+       anm.IMPLANT_NUMBER AS implantNumber,
+       anm.VENDOR_ANIMAL_NUMBER AS vendorNumber,
+       anm.ACTIVE_YN AS activeStatus,
+       st.status AS calculated_status,
+       alt.NAME AS damSire,
+       COALESCE(MAX(CAST(adt.CHANGE_DATETIME AS TIMESTAMP)), to_date('01/01/1970' ,'MM/DD/YYYY')) AS modified
+FROM Animal anm
+LEFT JOIN AUDIT_TRAIL adt ON anm.ANIMAL_ID_NUMBER = substring(PRIMARY_KEY_VALUES, length('ANIMAL_ID = '))
+AND adt.TABLE_NAME = 'ANIMAL' AND adt.COLUMN_NAME != 'DELETE'
```

Comment:
I think this will join to both inserted and modified rows in the audit table for the same animal id number.

Reply:
That's the intent.  The most recent insert or modified will supply the modified date for the row.  As you mentioned above I don't need to filter out 'DELETE' audit rows. That is fixed. Let me know if there is anything I'm missing on this.